website/docs/cloud-docs/api-docs/team-tokens.mdx

```diff
@@ -229,3 +229,71 @@ This endpoint supports pagination [with standard URL query parameters](/terrafor
   ]
 }
 ```
+
+## Show a Team Token
+
+A [Team Token](/terraform/cloud-docs/users-teams-organizations/teams#api-tokens) can be displayed for a particular team:
```

Comment:
Quick suggestions for active voice and capitalization (we only capitalize product names):
```suggestion
You can display a [team token](/terraform/cloud-docs/users-teams-organizations/teams#api-tokens) for a particular team using:
```